### PR TITLE
add items to tracklist in chunks to (hopefully) avoid timeouts

### PR DESCRIPTION
**Pull request by svoynow**

### CODE REVIEW

File: lib/hedwig_mopidy/responders/mopidy.ex [hunk 1]
```diff
@@ -238,6 +239,13 @@ defmodule HedwigMopidy.Responders.Mopidy do
     send message, response
   end
 
+  def add_tracks_in_batches(tracks, batch_size \\ 100) do
+    tracks
+    |> Enum.chunk(batch_size, batch_size, [])
```
Comment (bucchere) [inline lib/hedwig_mopidy/responders/mopidy.ex:244]: Shoudn't that be `Enum.chunk(tracks, batch_size, [])`?

File: lib/hedwig_mopidy/responders/mopidy.ex [hunk 2]
```diff
@@ -102,10 +102,11 @@ defmodule HedwigMopidy.Responders.Mopidy do
           playlist
         end
       end
+    send message, "Working on it.."      
     response =
       with  {:ok, :success} <- Tracklist.clear,
-            {:ok, playlist_refs} <- Playlists.get_items(playlist.uri),
-            {:ok, tracks} when is_list(tracks) <- Tracklist.add(playlist_refs |> Enum.map(fn(%Ref{} = r) -> r.uri end)),
+            {:ok, playlist_refs} <- Playlists.get_items(playlist.uri),
```
Comment (bucchere) [inline lib/hedwig_mopidy/responders/mopidy.ex:108]: This might also time out

File: lib/hedwig_mopidy/responders/mopidy.ex (hunk 1 above)
Comment (svoynow) [reply lib/hedwig_mopidy/responders/mopidy.ex:244]: unfortunately not, at least as I read the docs: https://hexdocs.pm/elixir/Enum.html#chunk/4

if you want to specify `leftover` you need the 3rd parameter which doesn't have a default value in the /4 case. Kind of awkward.

File: lib/hedwig_mopidy/responders/mopidy.ex (hunk 2 above)
Comment (svoynow) [reply lib/hedwig_mopidy/responders/mopidy.ex:108]: yeah, but we'll cross that bridge if/when we come to it.
Think of this as an experiment to see if the other one is the real problem.

File: lib/hedwig_mopidy/responders/mopidy.ex (hunk 1 above)
Comment (bucchere) [reply lib/hedwig_mopidy/responders/mopidy.ex:244]: You're right. I was misunderstanding how the pipe operator works.

File: lib/hedwig_mopidy/responders/mopidy.ex (hunk 2 above)
Comment (bucchere) [reply lib/hedwig_mopidy/responders/mopidy.ex:108]: Good deal. Ship it.

**Pull request merged**